Dockerfile

```diff
@@ -30,6 +31,11 @@ COPY --from=build /app/run.sh /app/run.sh
 COPY --from=build --chmod=0755 /app/entrypoint.sh /app/entrypoint.sh
 WORKDIR /app
 
+# Get browser binaries
+RUN /app/.pixi/envs/default/bin/playwright install --with-deps
+RUN /app/.pixi/envs/default/bin/rebrowser_playwright install --with-deps
+RUN /app/.pixi/envs/default/bin/camoufox fetch
```

Comment:
@castelao If I understand correctly, the code is set up to run this command the first time you execute it, if you hadn't run it before. I suspect this is why your image size changed after you created it!

Reply:
Now that we run it as part of setup, hopefully the container size will stay fixed.